README.md

```diff
@@ -58,7 +58,7 @@ end
 
 ## Events
 
-The IRC adapter will trigger the `:connected` and `:disconnected` events when the robot has connected and disconnected from IRC, respectively. There is no payload data for either event.
+The IRC adapter will trigger the `:connected`, `:disconnected` and `:user_joined_room` events when the robot has connected and disconnected from IRC, respectively. There is no payload data for either event.
```

Comment:
This sentence doesn't make sense with this change. `:user_joined_room` should probably be listed in a separate paragraph since it does have a payload.